### DISCUSSION TITLE
Environnent variable reset after some time.

### DISCUSSION THREAD

**Discussion by Romaindcrl** [deta/cloud-docs#648] [Help]
I'm running a python script that, on each run, should add 1 to an environment variable named "Counter". If I set Deta's Cron to 1 minute, everything is fine, but if I set it to 1 hour, the environment variable automatically resets to 0 between each execution of the script.

**Comment by lemonyte** [comment]
AWS Lambda environment variables do not persist for very long, see [this answer](https://stackoverflow.com/a/58533949/16237543) on Stack Overflow. You can easily use Base to store a counter field and [increment](https://docs.deta.sh/docs/base/sdk#update-operations) it.
